cmd/etlprov/README.md

```diff
@@ -8,38 +12,32 @@ $ pedsnet-etlprov [-model <model>] [-version <version>] [-truncate=false] [-igno
 
 Option | Description
 ---|---
--model <model> | Specify a data model other than 'pedsnet', e.g. 'i2b2_pedsnet'
--version <version> | Specify PEDSnet CDM version, e.g. 2.0.0 or 2.1.0
+-model <model> | Specify a data model other than the default of 'pedsnet', e.g. 'i2b2_pedsnet'
+-version <version> | Specify a model version other than the default of '2.0.0', e.g. 2.1.0 or 2.2.0
 -truncate=false | Show all errors, even redundant or excessive ones
--service <service> | Specify model service other than http://data-models.origins.link (not useful unless you run your own model service)
+-service <service> | Specify a model service other than http://data-models.origins.link (not useful unless you run your own model service)
 -ignore <entities> | Ignore a comma-separated list of entities (not normally used)
 -v | Show the version of this tool
 
-
 ### Example
 
 ```bash
-$ pedsnet-etlprov -model i2b2_pedsnet -version 2.0.0 ./files
-Validating against model 'i2b2_pedsnet/2.0.0'
-Scanning files in '/path/to/files/'
+$ pedsnet-etlprov -version 2.2.0 -truncate=false
+Validating against model 'pedsnet/2.1.0'
```

Comment:
You just used `-version 2.2.0` but the program is validating against `pedsnet/2.1.0`?